integration-tests/path-prefix/cypress/integration/path-prefix.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-undef */
+const { pathPrefix } = require('../../gatsby-config')
+
+const withTrailingSlash = url => `${url}/`
+
+describe(`Production pathPrefix`, () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it(`returns 200 on base route`, () => {
+    cy.location('pathname').should('eq', withTrailingSlash(pathPrefix))
+  })
+
+  describe('navigation', () => {
+    const pageTwoLink = `a[data-test-id="page-2-link"]`
```

Comment:
`getTestElement` (custom Gatsby command) might be better here

Reply:
Sure! Hard to do that when the PR isn't merged, so I just merged it :)